egs/madcat_ar/v1/local/create_line_image_from_page_image.py

```diff
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+# Minimum Bounding Box part in this script is originally (https://github.com/BebeSparkelSparkel/MinimumBoundingBox)
+
+
+import argparse
+import os
+import xml.dom.minidom as minidom
+from PIL import Image
+import numpy as np
+from scipy.misc import toimage
+
+from scipy.spatial import ConvexHull
+from math import sqrt
+from math import atan2, cos, sin, pi, degrees
+from collections import namedtuple
+
+parser = argparse.ArgumentParser(description="""Creates line images from page image.""")
+parser.add_argument('database_path1', type=str,
+                    help='Path to the downloaded (and extracted) mdacat data file 1')
```

Comment:
i think you mean madcat, not mdacat.

Please add an example of usage of this program.  Below is an example from an rnnlm script:
```
parser = argparse.ArgumentParser(description="This script get a vocab from unigram counts "
                                 "of words produced by get_unigram_counts.sh",
                                 epilog="E.g. " + sys.argv[0] + " data/rnnlm/data > data/rnnlm/vocab/words.txt",
                                 formatter_class=argparse.ArgumentDefaultsHelpFormatter)
```
And I think instead of 'data file 1' you mean data directory 1.